### ISSUE TITLE
MI requires REI to be installed

### ISSUE THREAD

**Issue by EntityinArray** [AztechMC/Modern-Industrialization#430]
Modern Industrialization requires REI to be installed in order to work. This requirement is unnecessary and doesn't allow Minecraft to launch with other recipe viewers, such as EMI for example.
```
Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'Modern Industrialization' (modern_industrialization) 1.2.3 with any version that is compatible with:
		 - Other constraints that can't be automatically determined
Unmet dependency listing:
	 - Mod 'Modern Industrialization' (modern_industrialization) 1.2.3 requires any version of roughlyenoughitems, which is missing!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
```

**Comment by Technici4n** [comment]
This was addressed in 1.19, and won't be changed in 1.18.

**Comment by EntityinArray** [comment]
Oh, i see, thank you for quick response
